# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserChild.h
# Commit: e1fd1fa67a65
# Full Hash: e1fd1fa67a6554ae8dc206a6fed9e5405a0bf778
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2021-04-01 03:12:48
# Regressor Bug: 1699721
# File Overlap Count: 1
# Description:
#   Bug 1699721 - Part 2: Track BrowserParent lifecycles during process switches, r=kmag
#   
#   This patch contains a large number of changes around the process switching
#   mechanism in order to avoid issues which are caused by a mismatched
#   understanding of the state of the process switch between processes in the
# ==============================================================================

diff -r 2b3a6042d48e -r e1fd1fa67a65 dom/ipc/BrowserChild.h
--- a/dom/ipc/BrowserChild.h	Wed Mar 31 16:51:58 2021 +0000
+++ b/dom/ipc/BrowserChild.h	Wed Mar 31 16:51:58 2021 +0000
@@ -558,8 +558,7 @@
   mozilla::ipc::IPCResult RecvUpdateNativeWindowHandle(
       const uintptr_t& aNewHandle);
 
-  mozilla::ipc::IPCResult RecvWillChangeProcess(
-      WillChangeProcessResolver&& aResolve);
+  mozilla::ipc::IPCResult RecvWillChangeProcess();
   /**
    * Native widget remoting protocol for use with windowed plugins with e10s.
    */